### ISSUE TITLE
Show only the first instance of a recurring event

### ISSUE THREAD

**Issue by nicooo** [plusonelabs/calendar-widget#46]
The first upcoming instance.

**Comment by yvolk** [comment]
This "bug" is an opposite of the #191 "bug": "Google events that repeat not showing" - it was "fixed" in April 2015.
The point is that different people need different behaviour. I also find that e.g. a monthly event-reminder "pay my bills" should not be shown more than once. Otherwise I will never see a single event scheduled for 5 months ahead - it will be lost in several duplicates of such "periodic reminders".

I would reformulate the new option though as: "Show only closest instance of recurring event" meaning that if e.g. I configured the Calendar Widget to "Show events that ended recently" and the closest instance of the monthly event occurred yesterday then the Calendar Widget should show that yesterday's past event instance, and not the next one, which will occur in 29 days. This will mean that I should check if I did what I needed to do _yesterday_ ...
